### PR TITLE
Child documents for citizen

### PR DESCRIPTION
**Pull request by Joosakur**

### CODE REVIEW

File: service/src/main/kotlin/fi/espoo/evaka/document/childdocument/ChildDocumentControllerCitizen.kt [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        clock: EvakaClock,
+        @PathVariable documentId: ChildDocumentId
+    ) {
+        return db.connect { dbc ->
+                dbc.transaction { tx ->
+                    accessControl.requirePermissionFor(
+                        tx,
+                        user,
+                        clock,
+                        Action.Citizen.ChildDocument.READ,
```
Comment (reynders) [inline service/src/main/kotlin/fi/espoo/evaka/document/childdocument/ChildDocumentControllerCitizen.kt:86]: I assume this is intentional reuse

File: service/src/main/kotlin/fi/espoo/evaka/shared/security/Action.kt [hunk 2]
```diff
@@ -437,6 +437,20 @@ sealed interface Action {
             READ_ATTENDANCE_SUMMARY(
                 IsCitizen(allowWeakLogin = true).guardianOfChild(),
                 IsCitizen(allowWeakLogin = true).fosterParentOfChild()
+            ),
+            READ_CHILD_DOCUMENTS(
+                IsCitizen(allowWeakLogin = true).guardianOfChild(),
```
Comment (reynders) [inline service/src/main/kotlin/fi/espoo/evaka/shared/security/Action.kt:442]: Is weak login checked to be enough for pedagogic docs?

**Review by reynders — APPROVED** [review]

File: service/src/main/kotlin/fi/espoo/evaka/shared/security/Action.kt (hunk 2 above)
Comment (Joosakur) [reply service/src/main/kotlin/fi/espoo/evaka/shared/security/Action.kt:442]: 😱 probably not, thanks!

Comment (Joosakur) [reply service/src/main/kotlin/fi/espoo/evaka/shared/security/Action.kt]: fixed

**Pull request merged**